src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -1,119 +1,33 @@
 import React from "react"
 import { AppContainer } from "v2/Apps/Components/AppContainer"
 import { createFragmentContainer, graphql } from "react-relay"
-import { AuctionsApp_currentSales } from "v2/__generated__/AuctionsApp_currentSales.graphql"
-import { AuctionsApp_pastSales } from "v2/__generated__/AuctionsApp_pastSales.graphql"
 import { AuctionsApp_me } from "v2/__generated__/AuctionsApp_me.graphql"
-import { Box, Text, Flex, Separator, Image, Join, Spacer } from "@artsy/palette"
+import { Box } from "@artsy/palette"
 import { AuctionsMeta } from "./Components/AuctionsMeta"
 import { HorizontalPadding } from "../Components/HorizontalPadding"
-import { SaleItemFragmentContainer } from "./Components/SaleItem"
 import { Footer } from "v2/Components/Footer"
-import { RouterLink } from "v2/Artsy/Router/RouterLink"
-import styled from "styled-components"
-import { MyBidsFragmentContainer } from "./Components/MyBids"
+import { RouteTabs, RouteTab } from "v2/Components/RouteTabs"
 
 export interface AuctionsAppProps {
   me: AuctionsApp_me
-  currentSales: AuctionsApp_currentSales
-  pastSales: AuctionsApp_pastSales
 }
 
-const AuctionsApp: React.FC<AuctionsAppProps> = ({
-  children,
-  currentSales,
-  pastSales,
-  me,
-}) => {
+const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
+
       <HorizontalPadding mt={4}>
-        <Flex justifyContent="space-between">
-          <Box width={["100%", "70%"]} pr={[0, 6]}>
-            <Text variant="largeTitle">Current Auctions</Text>
-            <Separator mt={1} mb={3} />
-            {currentSales.edges.map(({ node }, index) => {
-              return (
-                <Box my={4} key={index}>
-                  <SaleItemFragmentContainer sale={node} />
-                </Box>
-              )
-            })}
+        <RouteTabs mb={2}>
+          <RouteTab exact to="/auctions2">
+            Current Auctions
+          </RouteTab>
+          <RouteTab to="/auctions2/upcoming">Upcoming</RouteTab>
+          <RouteTab to="/auctions2/past">Past</RouteTab>
+        </RouteTabs>
```

Comment:
These `RouteTabs` / `RouteTab` components will tap into our framework router allowing us to jump between pages, while piggybacking on our [Tabs component](https://palette.artsy.net/elements/navigation/tabs/)